### PR TITLE
Updated TestContext information

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: docs/articles/nunit/writing-tests/attributes/testcase.md
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/attributes/testcase.md:41]:
```suggestion
* **TestOf** specifies the Type that this test is testing (this is not used within NUnit during test execution, but may serve a purpose for the test author)
```

Minor wording suggestion (assuming I got your intent correct)

File: docs/articles/nunit/writing-tests/TestContext.md
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/TestContext.md:156]:
```suggestion
* **Type** - The `TypeInfo` of the test
```

I think this is correct? (if so, it'll let you remove the cSpell dictionary addition as well I think

File: docs/articles/nunit/writing-tests/TestContext.md
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/TestContext.md:160]:
```suggestion
* **Properties** - An `IPropertyBag` of the test properties. Note: The `Get` method ONLY returns the first property
```

**Review by SeanKilleen — APPROVED** [review]
Generally LGTM -- a few questions but once addressed I'm fine if this gets merged 👍

**Comment by OsirisTerje** [comment]
Thanks @SeanKilleen !!

**Comment by OsirisTerje** [comment]
Perhaps we should add links to the generated API docs where relevant

**Comment by SeanKilleen** [comment]
@OsirisTerje great idea on referencing the API docs -- especially once I get https://github.com/nunit/docs/issues/941 completed, which should ensure the reference version of NUnit stays up to date in the docs.

**Pull request merged**
